test/unit-tests/Spectacle/Syntax/Modal.hs

```diff
@@ -0,0 +1,116 @@
+{-# LANGUAGE MultiWayIf #-}
+
+module Spectacle.Syntax.Modal
+  ( tests,
+  )
+where
+
+import Control.Arrow
+import Control.Monad
+import Data.Function ((&))
+import Hedgehog
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Range as Range
+import Test.Tasty
+import Test.Tasty.Hedgehog
+
+import Data.Type.Rec
+import Language.Spectacle.Exception.RuntimeException
+import Language.Spectacle.Lang
+import Language.Spectacle.Syntax.Error
+import Language.Spectacle.Syntax.Logic
+import Language.Spectacle.Syntax.Modal
+import Language.Spectacle.Syntax.Modal.Internal
+import Language.Spectacle.Syntax.NonDet
+import Language.Spectacle.Syntax.Plain
+import Language.Spectacle.Syntax.Prime
```

Comment:
The usual suggestion to qualify and/or explicitly list imports.

---

Also, I'm curious what you expect the user experience to be re: importing modules? Like if I'm a user using `spectacle` in anger, will I be able to do something like:

```haskell
import qualified Language.Spectacle as Spectacle
```

...and get everything I need without conflicting names? Or maybe I'll get most things, and then particular effects are imported explicitly:

```haskell
import Language.Spectacle.Syntax.NonDet (NonDet)
import qualified Language.Spectacle as Spectacle
import qualified Language.Spectacle.Syntax.NonDet as NonDet
```

Or maybe you could provide a `Prelude` module like some other libraries do (e.g. `Streamly.Prelude`) to go alongside specific effects.

This is kind of a vague / open-ended question, so don't feel like you need to have an answer or anything. Just something I'm curious about; something to think about maybe.

Reply:
Oh whoops, sorry about the imports. There will certainly be top-level modules which export all of the syntax and model checker functions but I haven't bothered including it yet, mostly because managing namespace while development is still so volatile is very tedious.

Reply:
Sounds good, thanks for explaining 👍